server_src/routes/settings.js

```diff
@@ -131,12 +131,14 @@ router.get("/server/restart", ensureAuthenticated, async (req, res) => {
   }
   res.send(serviceRestarted);
 });
+// left as a get request because I'm not sending any data, to me, I'm "getting" the update, but I can understand the "posting a command", can be made post just didn't see the need. Are there any rules to follow around this?
```

Comment:
GET = request data, very easy for browser
POST = action on data, doesnt explicitly have to return anything except 200/204 code
PATCH,PUT = actions like POST
DELETE = delete action

some other methods which nobody really cares about

Reply:
I'll change to post no drama, thanks for the explanation